controller/backing_image_controller.go

```diff
@@ -287,30 +287,34 @@ func (bic *BackingImageController) syncBackingImage(key string) (err error) {
 	return nil
 }
 
-func (bic *BackingImageController) IsBackingImageDataSourceCleanup(bi *longhorn.BackingImage) (finished bool, err error) {
+func (bic *BackingImageController) IsBackingImageDataSourceCleaned(bi *longhorn.BackingImage) (cleaned bool, requested bool, err error) {
```

Comment:
Can rename the returned variables. `(cleaned bool, requested bool, err requestError)`

Reply:
lol you are right, just typo.